src/utils/convert/attributes.js

```diff
@@ -0,0 +1,264 @@
+const svgAttributes = {
```

Comment:
I think you may remove any entries on this object in which the key is the same as the value (doesn't need to be converted), because you are doing this: https://github.com/moxystudio/react-svg/pull/6/files#diff-4b82f0129fea06b43e282c623952b225e02b1a4db9d49532c9f42750198653cdR9

Reply:
It makes sense since we are not validating if the attribute is valid for SVGs. I fixed it.